packages/belos/tpetra/test/BlockGmres/test_bl_gmres_hb_df.cpp

```diff
@@ -89,10 +88,9 @@ int mptestnumimages = 1;
 ///////////////////////////////////////////////////////////////////////////
 ///////////////////////////////////////////////////////////////////////////
 template <class Scalar> 
-RCP<LinearProblem<Scalar,MultiVector<Scalar,int>,Operator<Scalar,int> > > buildProblem()
+RCP<LinearProblem<Scalar,MultiVector<Scalar,int>,Operator<Scalar,int> > > buildProblem(const RCP<Map<int> >& vmap)
```

Comment:
The `using namespace Belos;` above makes this ambiguous.  `Belos::Operator` has one template parameter; `Tpetra::Operator` can take up to four.  I would recommend writing this:
```c++
template<class Scalar>
RCP<LinearProblem<Scalar, Tpetra::MultiVector<Scalar>, Tpetra::Operator<Scalar>>> buildProblem (const RCP<const Tpetra::Map<>>& vmap);
```

Reply:
@mhoemmen Yeah, I saw this locally using another compiler.  The changes have been made, I'm just testing locally before updating the pull request.  Thanks!